ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -193,6 +201,51 @@ export default class AlertTableRow extends React.Component {
     );
   };
 
+  showTags = (id, tags) => {
+    return tags.map((item) => (
+      <Badge color="light" key={`${item}`} data-testid={`alert-${id}-tags`}>
+        {item}
+      </Badge>
+    ));
+  };
+
+  getTags = (alert) => {
+    const { displayTags } = this.state;
```

Comment:
I'm a bit confused by the `displayTags` var name. I had the impression there's a toggle for the entire `Tag` column, not for the other 2+ tags that may be available.

Could we rename so it's more descriptive?